tests/functions/test_update_where.py

```diff
@@ -16,3 +24,50 @@ def test_update_where(dataframe):
         ),
         dataframe.replace("Cambridge", "Durham"),
     )
+
+
+def test_update_where_str():
+    """
+    Test that function works with pandas query-style string expression
+    """
+    df = pd.DataFrame(
+        {"a": [1, 2, 3, 4], "b": [5, 6, 7, 8], "c": [0, 0, 0, 0]}
+    )
+    expected = pd.DataFrame(
+        {"a": [1, 2, 3, 4], "b": [5, 6, 7, 8], "c": [0, 0, 10, 0]}
+    )
+    result = update_where(
+        df, conditions="a > 2 and b < 8", target_column_name="c", target_val=10
+    )
+
+    assert_frame_equal(result, expected)
+
+
+def test_update_where_2():
```

Comment:
I can't tell - is this test assessing something different than the other one? I'm not sure whether adding `NaN`s to the data in this particular case makes it unique rather than just a duplicate of the other test with different data. If there's no additional functionality "coverage" here, I'm not sure it's necessary to include.